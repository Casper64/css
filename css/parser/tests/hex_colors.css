
.dim {
    color: #;
    color: #abcdef;
    color: #123456;
    color: #7890j0;
    color: #zzxxzz;
}