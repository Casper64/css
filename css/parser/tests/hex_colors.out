css/parser/tests/hex_colors.css:3:12: error: expecting a hex color
    1 | 
    2 | .dim {
    3 |     color: #;
      |            ^
    4 |     color: #abcdef;
    5 |     color: #123456;
css/parser/tests/hex_colors.css:6:12: error: invalid hex color
    4 |     color: #abcdef;
    5 |     color: #123456;
    6 |     color: #7890j0;
      |            ~~~~~~~
    7 |     color: #zzxxzz;
    8 | }
css/parser/tests/hex_colors.css:7:12: error: invalid hex color
    5 |     color: #123456;
    6 |     color: #7890j0;
    7 |     color: #zzxxzz;
      |            ~~~~~~~
    8 | }